samples.bluemix/TestRead/TestRead.spl

```diff
@@ -9,13 +9,15 @@ composite TestRead {
 	stream<rstring line> FileContents = WebHdfsRead() {
 	param 
 	url:getSubmissionTimeValue("url");
-file: getSubmissionTimeValue("linefile","/tmp/TxtFile3.txt");
+	file: getSubmissionTimeValue("personfile","/tmp/People.txt");
 	password: getSubmissionTimeValue("password");
 	username: getSubmissionTimeValue("username");
 	}
 
 () as sink = FileSink(FileContents) {
-param file: "lines.txt";
+	param
+	file: "lines.txt";
+	quoteStrings:false;
```

Comment:
Instead of writing in csv format and setting quoteStrings to false, could we use line format?  That way, it'd be match what is being done by WebHdfsRead().  
````
() as sink = FileSink(FileContents) {
param 
file: "lines.txt";
format: line;
}